src/mixins/filing-mixin.ts

```diff
@@ -129,7 +128,6 @@ export default class FilingMixin extends Mixins(DateMixin) {
     const correctionFiling: CorrectionFilingIF = {
       header: {
         name: FilingTypes.CORRECTION,
-        certifiedBy: correctedFiling.header.certifiedBy,
```

Comment:
In both cases here, it's incorrect to set the Certified By as the original filing's certifier (ie, the correction may be certified by a different person, or no person if this is a staff-error correction).

The Edit UI should (and does) handle this case correctly.